src/Application.vala

```diff
@@ -158,6 +182,50 @@ public class AppCenter.App : Granite.Application {
             });
         }
     }
+
+    private void on_operation_finished (AppCenterCore.Package package, AppCenterCore.Package.State operation, Error? error) {
+        switch (operation) {
+            case AppCenterCore.Package.State.INSTALLING:
```

Comment:
It would be good to have an un-install version of this so that we can get a notification when things have been un-installed (or failed) in the background.